app/lib/constants/site_config.rb

```diff
@@ -113,8 +113,8 @@ module SiteConfig
         description: "The \"Client Secret\" portion of the GitHub Oauth Apps portal",
         placeholder: ""
       },
-      ga_view_id: {
```

Comment:
Was this just unused?

Reply:
Yeah, this was part of the functionality for using the Google Analytics API for tallying total page views, which we no longer use. So this is just dead functionality.